sdks/java/core/src/test/java/com/google/cloud/dataflow/sdk/runners/inprocess/InProcessCreateTest.java

```diff
@@ -127,7 +132,7 @@ public boolean equals(Object o) {
       return myString.equals(((UnserializableRecord) o).myString);
     }
 
-    static class UnserializableRecordCoder extends StandardCoder<UnserializableRecord> {
+    static class UnserializableRecordCoder extends AtomicCoder<UnserializableRecord> {
```

Comment:
It's not relevant in this context, but lets us delete some code (the coder is incidentally deterministic, fwiw)